js/bank.js

```diff
@@ -4,118 +4,113 @@ var rl = readline.createInterface({
   output: process.stdout
 });
 
-function start(balance) {
+
+function bank(balance) {
   this.balance = balance;
```

Comment:
Does this need to be `this` ? Or, can you use `var`?

Reply:
It needs to be this instead of var other wise the value of the balance returns undefined in the terminal.